### PR TITLE
Export to CSV now includes Gateway Language code and Gateway Language Quote

### PR DESCRIPTION
**Pull request by mannycolon**
## Describe what your pull request addresses (Please include relevant issue numbers):
- Export to CSV now includes Gateway Language code and Gateway Language Quote

### English GL
- Open a brand new project with English as the GL
- Make a selection, toggle the bookmark/reminder button, write a comment and make a verse edit/change.
- Then go back and export to CSV your CSV data for those checks should include the Gateway Language code and Gateway Language Quote in English.
### Hindi GL
- Open a brand new project with Hindi as the GL
- Make a selection, toggle the bookmark/reminder button, write a comment and make a verse edit/change.
- Then go back and export to CSV your CSV data for those checks should include the Gateway Language code and Gateway Language Quote in Hindi.

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/4012)

### CODE REVIEW

**Comment by PhotoNomad0** [comment]
Reviewed 20 of 20 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by RoyalSix** [comment]
Reviewed 17 of 20 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[src/js/helpers/csvHelpers.js, line 254 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012#-L7Q5mYzBUG4RNJdCpi4:-L7Q5mYzBUG4RNJdCpi5:b8palhq) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/7b86ff27772e485d7841bd33d8e50d7b1ed3382f/src/js/helpers/csvHelpers.js#L254)):*
> ```JavaScript
>     if (objectData && objectData.contextId && objectData.contextId.tool && objectData.contextId.groupId) {
>       const toolName = objectData.contextId.tool;
>       groupsIndex = groupsIndexHelpers.getGroupsIndex('en', toolName);
> ```

This should always be 'en' ? 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by RoyalSix** [comment]
Reviewed 3 of 20 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by RoyalSix** [comment]
Local Testing: `LGTM`
Code Review: One comment.

**Comment by richmahn** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[src/js/helpers/csvHelpers.js, line 254 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012#-L7Q5mYzBUG4RNJdCpi4:-L7Q81yUD-JMDxQwnh0l:b-mx87al) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/7b86ff27772e485d7841bd33d8e50d7b1ed3382f/src/js/helpers/csvHelpers.js#L254)):*
<details><summary><i>Previously, RoyalSix (Sterling (Jay) Scott) wrote…</i></summary><blockquote>

This should always be 'en' ? 
</blockquote></details>

Yeah, this probably instead be `LanguagesHelpers.DEFAULT_GATEWAY_LANGUAGE`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by richmahn** [comment]
Reviewed 20 of 20 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by mannycolon** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[src/js/helpers/csvHelpers.js, line 254 at r1](https://beta.reviewable.io/reviews/unfoldingword-dev/translationcore/4012#-L7Q5mYzBUG4RNJdCpi4:-L7QlbDx5rgv2VFTLUnG:b-2imal4) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/7b86ff27772e485d7841bd33d8e50d7b1ed3382f/src/js/helpers/csvHelpers.js#L254)):*
<details><summary><i>Previously, richmahn (Richard Mahn) wrote…</i></summary><blockquote>

Yeah, this probably instead be `LanguagesHelpers.DEFAULT_GATEWAY_LANGUAGE`
</blockquote></details>

@richmahn @royalsix If it is undefined it will always be English because now going forward it will have the GL in it which make sit defined therefore the only one undefined means it was generated with English.
@richmahn No need for that

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by mannycolon** [comment]
Review status: 18 of 20 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/js/helpers/csvHelpers.js, line 254 at r1](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012#-L7Q5mYzBUG4RNJdCpi4:-L7Qo9gIJc_hcaXOLAuI:ba7gkx5) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/7b86ff27772e485d7841bd33d8e50d7b1ed3382f/src/js/helpers/csvHelpers.js#L254)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

@richmahn @royalsix If it is undefined it will always be English because now going forward it will have the GL in it which make sit defined therefore the only one undefined means it was generated with English.
@richmahn No need for that
</blockquote></details>

In other words, checks that were generated and dont have the GL information mean that they were generated in the past when we didnt persist the GL with the check.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by richmahn** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012)*

**Comment by richmahn** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/4012#-:-L7R19FE1oEhuEuspJ8z:bnfp4nl)*
